internal/runner/rmq.go

```diff
@@ -149,8 +154,13 @@ func (rmq *RabbitMQ) attachQ(name string) (conn *amqp.Connection, ch *amqp.Chann
 		return nil, nil, kv.Wrap(errGo).With("stack", stack.Trace().TrimRuntime()).With("uri", rmq.Identity)
 	}
 
+	if errGo = ch.Confirm(false); errGo != nil {
+		return nil, nil, kv.Wrap(errGo).With("stack", stack.Trace().TrimRuntime()).With("uri", rmq.Identity)
+	}
+
```

Comment:
So what this is for?

Reply:
This ensure thats the transmit was completed before continuing. https://pkg.go.dev/github.com/streadway/amqp?utm_source=godoc#Channel.Confirm.  I added it as there were performance issues with RMQ servers in our data center and this might have an effect on pending notifications and acks.